fulltext/backends/__eml.py

```diff
@@ -0,0 +1,16 @@
+from email import message_from_file
+
+import codecs
+
+from six import StringIO
+
+
+def _get_file(f, **kwargs):
+    text, f = StringIO(), codecs.getreader('utf8')(f, errors='ignore')
+    m = message_from_file(f)
+
+    for part in m.walk():
+        if part.get_content_type().startswith('text/plain'):
+            text.write(part.get_payload())
+
+    return text.getvalue()
```

Comment:
file `f` should be closed

Reply:
`f` belongs to caller. I am wrapping it in a `StreamReader`, but if I close _that_ it will close `f` too.

Reply:
oh you're right